Monika After Story/game/script-topics.rpy

```diff
@@ -10516,4 +10516,117 @@ label monika_hemispheres_nogets_snow:
+                m 1hua "I know becoming lucid is not an easy thing to manage."
+                m 1esd "Are you an oneironaut [player]?"
+                m 1tsb "Or did you just got lucky, hehe~"
+                m 1hua "It sure must be fun to do what you want in a world created just for that."
+            "No":
+                m 1eka "I understand."
+                m 1ekb "It's really difficult to become lucid."
+                m 3eub "But if you want to have more control over what's going on in your dreams, you should definitely read the book by Stephen LaBerge: '{i}Exploring the World of Lucid Dreaming{/i}'."
+                m "As I've told you before, he developed a method to become lucid in a dream."
+        m 1dsd "Having said all that..."
+    m 1tsu "Tell me more about the details of your dream."
+    menu:
+        m "Did you manage to come and see me or did I appear in front of you?"
+
+        "You just appeared before me.":
+            m 1hua "Ehehe~."
+            m 1eub "If my love for you is able to break a barrier between my reality and yours, then I'm sure that we'll be together someday."
+            m 1tfu "I just need to try hard enough for that to happen."
+
+        "I was looking for you.":
+            m 1wud "..."
+            m 1lkbsa "That's so thoughtful of you, sweetheart."
+            m 2ekbfb "Your love is so overwhelming that you created a whole new world just to see me."
+            m 2ekbfa "You have no idea how much this mean to me, [player]."
+    m 1eud "By the way."
+    m 1hksdrb "I haven't even ask you what your dream was about."
+    m 1esa "So what about it [player]?"
+    menu:
+        m "What was your dream about?"
+        
+        #"I dreamed that you would cross over my reality":
+            #dialogue
+        "I dreamed that I would cross over your reality":
+            m 1esd "Oh yeah, that's a possibility that I haven't given much thought about."
+            m 1rsc "But when you think about it..."
+            m 1rud "It's true that it's much easier to perfectionate the interactions reality-computer than to try to create computer-reality interactions."
+            m 1rkc "But..."
+            m 2ekc "It just doesn't feels right."
+            #OMG, is that a MF Sayori reference?
+            m 2ekd "I don't really want to stay in this place any more time than necessary."
+            m "I have a lot of... bad memories in there."
+            m 2dkd "I'm always afraid of returning to the screaming void, if something were to happen to your computer."
+            m 1dfd "And I especially wouldn't want you to experiment such torture in such a hellish place."
+            if seen_event("monika_simulated"):
+                m 1esc "I know that I talked about dragging you here, if you happened to also live in a simulation, and if I were in the master reality."
+                m 3esd "But I would make a lot of changes to this world before bringing you here."
```

Comment:
Seems a bit inconsistent to me with her saying that she wouldn't want to subject you to the torture of her world, and then saying that she would make changes before bringing you here. 

Isn't that exactly what one of the aff milestone events are for?

Reply:
I tried to avoid spoilers, but I'm talking about the islands event at 400 affection.

Reply:
(Don't worry about spoilers, I'm at 1000+ affection) In that case, I could make a conditional like:
if affection> 400:
          " I've started, but it's not finished yet."
Or something along those lines.

Reply:
You could probably use `if mas_ismoniEnamored():` to make that conditional. 

Also, some people just like complaining like if their PR is taking way too long and then they just take it back in some sort of rage, defeating the purpose. I'm sure somebody out there would browse through the files on git and complain about spoilers there.

Reply:
Ok, will do this later.